src/plugins/presentation_panel/public/panel_component/panel_header/presentation_panel_hover_actions.tsx

```diff
@@ -535,12 +540,12 @@ export const PresentationPanelHoverActions = ({
           ref={hoverActionsRef}
           className="embPanel__hoverActionsWrapper"
           css={css`
-            height: ${euiThemeVars.euiSizeXL};
+            height: ${euiTheme.size.XL};
             position: absolute;
             top: 0;
             display: flex;
             justify-content: space-between;
-            padding: 0 ${euiThemeVars.euiSize};
+            padding: 0 ${euiTheme.size.medium};
```

Comment:
Blocking `euiTheme.size.medium` does not exists. Based on the original value, this likely should be `euiTheme.size.base`.